clients/js/README.md

```diff
@@ -17,6 +17,7 @@ private keys, based on `.env.sample` in this folder.
 worm [command]
 
 Commands:
+  worm edit-vaa                                   Allows editing / creating / signing VAAs
```

Comment:
Nit: since you call out that verify-vaa only works on Ethereum, should you call it out here, too?

Reply:
It's not that these commands "only work on Ethereum". They are chain independent. They just access the core contract on Ethereum to get the guardian set and verify signatures. 

I will remove the reference to Ethereum from verify-vaa.